src/FormControl/test/FormControlSpec.tsx

```diff
@@ -254,8 +255,14 @@ describe('FormControl', () => {
     fireEvent.change(container.querySelector('#username') as HTMLInputElement, {
       target: { value: 'username' }
     });
-    assert.deepEqual(refValue, { username: 'username', email: '' });
-    assert.deepEqual(refError, { username: 'The length cannot exceed 2' });
+    fireEvent.change(container.querySelector('#password') as HTMLInputElement, {
+      target: { value: 'password' }
+    });
+    assert.deepEqual(refValue, { username: 'username', password: 'password', email: '' });
+    assert.deepEqual(refError, {
+      username: 'The length cannot exceed 2',
+      password: 'The length cannot exceed 2'
+    });
```

Comment:
```js
 expect(refValue).to.deep.equal({ username: 'username', password: 'password', email: '' })
```
We uniformly adjust the test assertion to [BDD](https://www.chaijs.com/api/bdd/) style.

Reply:
done